recipe/meta.yaml

```
@@ -1,44 +1,51 @@
 {% set name = "pytest-split" %}
-{% set version = "0.8.0" %}
+{% set version = "0.10.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8571a3f60ca8656c698ed86b0a3212bb9e79586ecb201daef9988c336ff0e6ff
+  url: https://github.com/jerry-git/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ea57791544b034bf4cfae57cf2d036835c4e5a97b0a796da03c974c93106b264
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<36 or py>311]
```

```suggestion
  skip: True  # [py<38]
```
https://github.com/jerry-git/pytest-split/blob/0.10.0/pyproject.toml#L36